### DISCUSSION TITLE
New device to add: Xiaomi Mi Sticks - possible?

### DISCUSSION THREAD

**Discussion by thehijacker** [PRProd/HA-Firemote#116] [General]
Hello,

Can you also add the remote controls for the Xiaomi Mi Sticks. I have the 4K model, but it also exists HD version.

https://www.mi.com/global/product/xiaomi-tv-stick-4k/

https://www.mi.com/global/product/mi-tv-stick/

I think remote control is the same.

Also I kindly ask to add the "T-2 TV" application in the list. It is the main application to watch TV channels from my ISP. URL to the Google Store page:

https://play.google.com/store/apps/details?id=tv.perception.clients.tv.android&hl=sl&gl=US

Thank you.

**Comment by PRProd** [comment]
Requests transferred to issue #117 and issue #118
